frontend/src/pages/journal/journal.tsx

```diff
@@ -37,23 +33,14 @@ const Journal: React.FC = () => {
     setIsSidebarShow(true);
   }, [setIsSidebarShow, navigate]);
 
-  const handleGetSelectedNote = useCallback(async () => {
-    await dispatch(journalActions.getAllJournalEntries(filter));
-
-    dispatch(journalActions.setSelectedJournalEntry(Number(id)));
-  }, [dispatch, filter, id]);
-
-  useEffect(() => {
-    void handleGetSelectedNote();
-  }, [handleGetSelectedNote, id]);
-
   return (
     <>
       <JournalSidebar
         isSidebarShown={isSidebarShown}
         onSetIsSidebarShow={setIsSidebarShow}
         filter={filter}
         onSetFilter={setFilter}
+        id={id}
```

Comment:
don't pass id, this should be passed in functions how it was before in this component
```suggestion
```

Reply:
done